services/core-api/app/api/mines/response_models.py

```diff
@@ -115,6 +122,7 @@ def format(self, value):
         'permit_status_code': fields.String,
                                                                                  # 'permit_status_code_description': fields.String,
         'permit_amendments': fields.List(fields.Nested(PERMIT_AMENDMENT_MODEL)),
+        'bonds':fields.List(fields.Nested(BOND_MODEL))
```

Comment:
What was the need for adding bond to the permit response model?

Reply:
this is used to restrict deletion of permits with bonds

Reply:
Ah, I see. For front-end validation.